### PR TITLE
[Term Entry] Kotlin Math .acos()

### LINKED ISSUE #3512: [Term Entry] Kotlin Math .acos()
### Reason/inspiration (optional)

We would like a new term entry on the `.acos()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/acos/acos.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by rahul0x00**
### Description
Added a new term entry for Kotlin .acos() method
Closes #3512 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/acos/acos.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+Title: '.acos()'
+Description: 'Calculates the arccosine or reverse cosine of a value, returning the angle in radians.'
```
Comment (rahul0x00) [inline content/kotlin/concepts/math-methods/terms/acos/acos.md:3]: should i also include inverse cosine?

Description: 'Calculates the arccosine or reverse cosine or inverse cosine of a value, returning the angle in radians.'

File: content/kotlin/concepts/math-methods/terms/acos/acos.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  - 'paths/computer-science'
+---
+
+The **`.acos()`** method in Kotlin's `Math` class is used to calculate the
+arccosine of a given value, returning the corresponding angle in radians.
+
+## Syntax
+
+```pseudo
+val angle = Math.acos(value)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/acos/acos.md:21]:
```suggestion
import kotlin.math.*
val angle = acos(value)
```

File: content/kotlin/concepts/math-methods/terms/acos/acos.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+`value`: The value for which you want to calculate the arccosine. It should be
+in the range [-1, 1].
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/acos/acos.md:25]:
```suggestion
`value`: The given value to be used in the arccosine calculation. The value should be in the range `-1` to `1`.
```

File: content/kotlin/concepts/math-methods/terms/acos/acos.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+The `.acos()` method returns a value of type `double` representing the angle in
+radians. The result is in the range [0, PI].
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/acos/acos.md:28]:
```suggestion
The `.acos()` method returns a value of type `double` representing the angle in radians. The result will be in the range `0` to `PI`.
```

File: content/kotlin/concepts/math-methods/terms/acos/acos.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Example
+
+This example shows how to use the `.acos()` method to calculate the arccosine of
+a value:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/acos/acos.md:33]:
```suggestion
This example shows how to use the `acos()` method to calculate the arccosine of a value:
```

File: content/kotlin/concepts/math-methods/terms/acos/acos.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+```kotlin
+import kotlin.math.*
+
+fun main() {
+    val value = 0.5
+    val angle = Math.acos(value)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/acos/acos.md:40]:
```suggestion
    val angle = acos(value)
```

File: content/kotlin/concepts/math-methods/terms/acos/acos.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    println("The arccosine of $value is $angle radians.")
+}
+```
+
+The output of this code will be:
+
+```shell
+1.0471975511965979 rad
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/acos/acos.md:48]: this output doesn't match the print statement

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@rahul0x00 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by rahul0x00** [comment]
@caupolicandiaz 
Hi, i made the suggested changes

**Review by caupolicandiaz — APPROVED** [review]
@rahul0x00 This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@rahul0x00 @caupolicandiaz Looks good.

**Pull request merged**